libs/rollup.config.js

```diff
@@ -23,43 +34,131 @@ const commonConfig = {
     resolve({ extensions, preferBuiltins: true }),
     typescript()
   ],
-  external: [
-    ...Object.keys(pkg.dependencies),
-    ...Object.keys(pkg.devDependencies),
-    'ethereumjs-util',
-    'ethereumjs-wallet',
-    'ethers/lib/utils',
-    'ethers/lib/index',
-    'hashids/cjs',
-  ]
+  external
+}
+
+// For the browser bundle, these need to be internal because they either:
+// * contain deps that need to be polyfilled via `nodePolyfills`
+// * are ignored via `ignore`
+const internal = [
+  'eth-sig-util',
+  'ethereumjs-tx',
+  'ethereumjs-util',
+  'ethereumjs-wallet',
+  'graceful-fs',
+  'node-localstorage',
+  'web3'
+]
+
+const browserConfig = {
+  plugins: [
+    ignore(['web3', 'graceful-fs', 'node-localstorage']),
+    resolve({ extensions, preferBuiltins: false }),
+    commonjs({
+      extensions,
+      transformMixedEsModules: true,
+      dynamicRequireTargets: [
+        'data-contracts/ABIs/*.json',
+        'eth-contracts/ABIs/*.json'
+      ]
+    }),
+    alias({
+      entries: [{ find: 'stream', replacement: 'stream-browserify' }]
```

Comment:
this is dope

Reply:
This is actually an interesting one bc the rollup node polyfill for `stream` has circular dependencies the break when subsequently bundled in webpack. So had to use a different polyfill essentially